src/parser/parsed_data/create_index_info.cpp

```diff
@@ -11,6 +13,70 @@ CreateIndexInfo::CreateIndexInfo(const duckdb::CreateIndexInfo &info)
       scan_types(info.scan_types), names(info.names) {
 }
 
+static void RemoveTableQualificationRecursive(unique_ptr<ParsedExpression> &expr, const string &table_name) {
+	if (expr->GetExpressionType() == ExpressionType::COLUMN_REF) {
+		auto &col_ref = expr->Cast<ColumnRefExpression>();
+		auto &col_names = col_ref.column_names;
+		if (col_ref.IsQualified() && col_ref.GetTableName() == table_name) {
+			col_names.erase(col_names.begin());
+		}
+	} else {
+		ParsedExpressionIterator::EnumerateChildren(*expr, [&table_name](unique_ptr<ParsedExpression> &child) {
+			RemoveTableQualificationRecursive(child, table_name);
+		});
+	}
+}
+
+string CreateIndexInfo::ToString() const {
+	string result;
+
+	result += "CREATE";
+	D_ASSERT(constraint_type == IndexConstraintType::UNIQUE || constraint_type == IndexConstraintType::NONE);
+	if (constraint_type == IndexConstraintType::UNIQUE) {
+		result += " UNIQUE";
+	}
+	result += " INDEX ";
+	// FIXME: 'CONCURRENTLY' ??
```

Comment:
@Mytherin do we care that CONCURRENTLY is lost in stringification?
on `main` we preserve this because `sql` still contains it

Reply:
CONCURRENTLY is a nop we only accept for postgres compat so no